recipes/hiredis/0.14.1/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+
+from os import rename
+from os.path import join
+
+
+class HiredisConan(ConanFile):
+    name = "hiredis"
+
+    description = "Minimalistic C client for Redis >= 1.2"
+    topics = "conan", "c", "redis"
+
+    license = "BSD 3-Clause \"New\" or \"Revised\" License"
+    settings = "os", "arch", "compiler", "build_type"
+
+    homepage = "https://github.com/redis/hiredis"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False]
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": True
```

Comment:
```suggestion
        "shared": False
```

Use static by default. That's a CCI convention